cfn/guardduty.template

```diff
@@ -97,8 +97,13 @@
                              ]
                             }
                             },
-                            "Action": "kms:*",
-                            "Resource": "*"
+                            "Action": [
+                                "kms:Decrypt",
+                                "kms:Encrypt"
+                            ],
+                            "Resource": {
+                                "Ref":"SecretKey"
```

Comment:
`SecretKey` is not an AWS resource. This is an AIMS secret. It can't be used here.

Reply:
I worried that was the case, been quite a while since I wrote CFN. How can I restrict KMS to only operate on that field then?

Reply:
I don't think it is possible.
Please try to find out if it is possible to restrict policy to specific key ARN and/or Lambda env resources.